Simplenote/NoticeView.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+protocol NoticePresentingDelegate {
+    func noticePressBegan()
+    func noticePressEnded()
+}
+
+class NoticeView: UIView {
+
+    // MARK: Properties
+    //
+    @IBOutlet weak var stackView: UIStackView!
+    @IBOutlet weak var noticeLabel: UILabel!
+    @IBOutlet weak var noticeButton: UIButton!
+
+    var delegate: NoticePresentingDelegate?
+    var action: (() -> Void)? {
+        didSet {
+            noticeButton.isHidden = action == nil
+        }
+    }
+
+    // MARK: Initialization
+    //
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setupView()
+    }
+
+    // MARK: View Layout
+    //
+    private func setupView() {
+        translatesAutoresizingMaskIntoConstraints = false
+        setupViewStyles()
+        setupLongPress()
+
+        layoutIfNeeded()
+    }
+
+    private func setupLongPress() {
+        let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(viewWasLongPressed(_:)))
+        addGestureRecognizer(longPressGesture)
+    }
+
+    private func setupViewStyles() {
+        stackView.backgroundColor = UIColor(lightColor: .spGray3, darkColor: .darkGray2)
```

Comment:
Nipticky: can we add a color in `UIColor+Studio`?  Maaaybe **simplenoteNoticeBackgroundColor** (or so)